lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -188,23 +189,33 @@ class Runner(
       val context = JVMOptionsContext(enginePackagePath = engine.path)
 
       val manifestOptions =
-        engine.defaultJVMOptions.filter(_.isRelevant).map(_.substitute(context))
+        if (nativeImage) Seq()
+        else
+          engine.defaultJVMOptions
+            .filter(_.isRelevant)
+            .map(_.substitute(context))
       val environmentOptions =
         jvmOptsFromEnvironment.map(_.split(' ').toIndexedSeq).getOrElse(Seq())
       val commandLineOptions = jvmSettings.jvmOptions.map(
         translateJVMOption(_, standardOption = true)
-      ) ++
-        jvmSettings.extraOptions.map(
-          translateJVMOption(_, standardOption = false)
-        )
+      ) ++ (if (nativeImage) Seq()
+            else
+              jvmSettings.extraOptions.map(
+                translateJVMOption(_, standardOption = false)
+              ))
       val componentPath = engine.componentDirPath.toAbsolutePath.normalize
+      val modulePathOptions =
+        if (nativeImage) Seq()
+        else
+          Seq(
+            "--module-path",
+            componentPath.toString,
+            "-m",
+            "org.enso.runner/org.enso.runner.Main"
```

Comment:
Once we rely on `bin/enso` native binaries, we will no longer be able to access `Main` directly. Such a _"JVM switch"_ will be (and already is) handled by `bin/enso` when `--jvm` option is passed to them. E.g. this code should disappear at the end, and project manager shall always call into `bin/enso`.